internals/overlord/servstate/manager_test.go

```diff
@@ -1763,3 +1763,62 @@ type fakeLogManager struct{}
 func (f fakeLogManager) ServiceStarted(serviceName string, logs *servicelog.RingBuffer) {
 	// no-op
 }
+
+func (s *S) TestNoWorkingDir(c *C) {
+	// Service command should run in current directory (package directory)
+	// if "working-dir" config option not set.
+	dir := c.MkDir()
+	err := os.Mkdir(filepath.Join(dir, "layers"), 0755)
+	c.Assert(err, IsNil)
+	manager, err := servstate.NewManager(s.st, s.runner, dir, nil, nil, fakeLogManager{})
+	c.Assert(err, IsNil)
+	defer manager.Stop()
+
+	outputPath := filepath.Join(dir, "output")
+	layer := parseLayer(c, 0, "layer", fmt.Sprintf(`
+services:
+    nowrkdir:
+        override: replace
+        command: /bin/sh -c "pwd >%s; sleep %g"
```

Comment:
So this doesn't depend on which directory you run `go test` from?

Reply:
No, `go test` is explicitly documented to run the binary for the test package in the package directory.